README.md

```diff
@@ -125,3 +130,103 @@ But first we must confirm that it is a topology.
   | Topology t -> Topology.foreign_members t;;
 - : (string * json) list = [("extra", `A [`String "Wow!"])]
 ```
+
+# Topojsone
+Topojson is a non-blocking, streaming parser for TopoJSON objects. It uses jsone and [Eio](https://github.com/ocaml-multicore/eio) library.
```

Comment:
We don't actually depend on Eio (except in the tests), you'll notice our source and destinations are actually just functions that fill and read buffers, it is up to the user to provide non-blocking versions of these functions in order for the parser to be truly non-blocking. I would maybe state that and then say one such library that can provide these functions is Eio and that is what we shall use for the rest of the example :))

Reply:
Okay.😊